src/main/kotlin/app/revanced/patcher/util/proxy/mutableTypes/MutableMethod.kt

```diff
@@ -13,7 +13,7 @@ class MutableMethod(method: Method) : Method, BaseMethodReference() {
     private var accessFlags = method.accessFlags
     private var returnType = method.returnType
 
-    // Create own mutable MethodImplementation (due to not being able to change members like register count)
+    // Create own mutable MethodImplementation (due to not being able to change members like register count).
```

Comment:
Should be classified as a `TODO`, otherwise it can be unclear to any future readers.
```suggestion
    // TODO: Create own mutable MethodImplementation (due to not being able to change members like register count).
```